client/components/duration/duration-filter.js

```diff
@@ -0,0 +1,49 @@
+angular.module("managerApp").filter("duration", [
+    "$translate",
+    "$filter",
+    ($translate, $filter) => {
+        const unitHash = {
+            m: "month",
+            d: "day",
+            j: "day",
+            y: "year",
+            a: "year"
+        };
+        const simpleDurationReg = /(^[0-9]+)([mdjya]?)$/;
```

Comment:
It looks really complicated for such a simple task, maybe you should ask people in charge of the API to give you data you can exploit directly without having to do whatever this is, is this possible ?

Reply:
Don't have time for that, but I'll look into it.